addons/crm/report/crm_activity_report_views.xml

```diff
@@ -28,7 +28,7 @@
             <field name="model">crm.activity.report</field>
             <field name="arch" type="xml">
                 <search string="Activities Analysis">
-                    <filter string="My Activities" name="my_activity" domain="[('user_id', '=', uid)]"/>
```

Comment:
It is renamed so that all activity filters have the same name. So we have to update existing context key.